src/libraries/System.Net.Quic/src/System.Net.Quic.csproj

```diff
@@ -76,15 +81,27 @@
     <Reference Include="System.Threading" />
     <Reference Include="System.Threading.Channels" />
   </ItemGroup>
+
   <!-- Support for deploying msquic -->
+  <ItemGroup Condition="'$(TargetsWindows)' == 'true' and
+                        ('$(TargetArchitecture)' == 'x64' or '$(TargetArchitecture)' == 'x86')">
+    <BinPlaceDir Include="$(MicrosoftNetCoreAppRuntimePackNativeDir)" ItemName="NativeBinPlaceItem" />
+    <BinPlaceDir Include="$(NetCoreAppCurrentTestHostSharedFrameworkPath)" ItemName="NativeBinPlaceItem" />
+    <BinPlaceDir Include="$(NetCoreAppCurrentRuntimePath)" ItemName="NativeBinPlaceItem" />
+    <NativeBinPlaceItem Include="$(PkgSystem_Net_MsQuic_Transport)\runtimes\win10-$(TargetArchitecture)\native\msquic.dll" TargetPath="MsQuic.Native.dll" />
+    <NativeBinPlaceItem Include="$(PkgSystem_Net_MsQuic_Transport)\runtimes\win10-$(TargetArchitecture)\native\msquic.pdb" TargetPath="MsQuic.Native.pdb" />
+  </ItemGroup>
+
   <ItemGroup>
     <Content Include="msquic.dll" Condition="Exists('msquic.dll')">
```

Comment:
Do we still need this?

Reply:
I was wondering about that and I decide to leave if dev testing until we fully embark on the new path.

Reply:
I think for dev testing it is now easy, you can build the msquic repo locally and just copy the native asset manually or use the locally generated nuget package.

Reply:
you convince me. I remove the two windows entries since we have better way now.